pages/signup.php

```diff
@@ -128,10 +138,15 @@
       <input type="text" name="email" placeholder="Email" class="requiredField">
       <input type="number" name="hosp_id" placeholder="Hospital ID" class="requiredField">
       <input type="password" name="pswd" placeholder="Password" id="pswd" class="requiredField">
-      <input type="password" name="re_pswd" placeholder="Re-enter Password" class="requiredField">
+      <input type="password" name="re_pswd" placeholder="Re-enter Password" id="pswd2" class="requiredField">
+      <div id="showpass">
+                <input type="checkbox" id="box" onclick="box1()">
+                <span id="notice">show password</span>
+            </div>
```

Comment:
Also, do you know how to use `phpfmt` to format PHP files. If you are using vscode as your IDE there is an extension for it

Reply:
I asked for formatting this file (`signup.php`) since the open and close `div` that you added was not proper.

Reply:
> I asked for formatting this file (`signup.php`) since the open and close `div` that you added was not aligned properly and the spacing was `4` instead of `2`.
> 
> I will take care of this if you can't do it.

i will format it manually